### PR TITLE
PR for branch 'styling' that adds styling to website

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Added styling to website through various css files located in project

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @jkanyi-web ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [x] Kindly remove the live demo section from your readme file since this project is not deployed. This is a general requirement for all projects.
- [x] Kindly add 1 to 3 potential features to the Future Features section of your readme file. This is a general requirement for all projects.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jkanyi-web** [comment]
Thanks for the review @1GORDON . The changes have been effected feel free to make another review.

**Comment by jkanyi-web** [comment]
In interest of time you can text me on whatsapp(+254728658873) incase there are any other changes. I'm also available on zoom and slack

File: README.md [hunk 1]
```diff
@@ -7,7 +7,6 @@
   - [🛠 Built With ](#-built-with-)
```
Comment (MeqdamAlqudah) [inline README.md:7]: - [x] Please the navigation and the logo should be algined on the same line as in the design 
- [x] Please the color of the navigation links doesn't match the design the link is underlined which doesn't match the design 
- [x] The profile icon doesn't match the one in the design there is  border around it in the design you should make sure it match the design
your design 
image
Design 
image

File: README.md [hunk 2]
```diff
@@ -7,7 +7,6 @@
...
     - [Key Features ](#key-features-)
```
Comment (MeqdamAlqudah) [inline README.md:8]: - [x] Please there should be a verticla border between the action links and buttons as in the design.
- [x] Please the color of the book card info doesn't match the design the book category should be in light gray color and the author color should be in blue , (please make sure you got the same colors)
image

File: README.md [hunk 3]
```diff
@@ -7,7 +7,6 @@
...
     - [Home page](#home-page)
```
Comment (MeqdamAlqudah) [inline README.md:9]: - [x] Please there is a border upper the books form make sure to implement it , 
Your design 
image
Design 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @jkanyi-web , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## Highlights
:+1: adding books
:+1: progress bar

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (jkanyi-web) [reply README.md:9]: @ [MeqdamAlqudah] why are you not considerate of my current time situation? Do you want me to fail my week?

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @jkanyi-web , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jkanyi-web** [comment]
Thanks for that Alexander. Changes have been made, awaiting review

**Review by topeogunleye — APPROVED** [review]
### Hi @jkanyi-web, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Made UI changes as suggested ✔️ 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Already Approved ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — COMMENTED** [review]
## Hi @jkanyi-web, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**